tests/test_features.py

```diff
@@ -0,0 +1,13 @@
+from cyclic_boosting import CBPoissonRegressor
+import numpy as np
+
+
+def test_poisson_regressor_feature_importance(get_inputs):
+    X, y, feature_prop, feature_groups = get_inputs
+    est = CBPoissonRegressor(
+        feature_groups=feature_groups,
+        feature_properties=feature_prop,
+    )
+    est.fit(X, y)
+    norm_feature_importances = est.get_feature_importances()
+    np.testing.assert_almost_equal(sum(norm_feature_importances), 1.0, 3)
```

Comment:
You should also check the individual feature importances.

Reply:
Yeah, this test - as it is - is more of a placeholder. I want to 1) check the individual feature importances, 2) check if feature A has importance X, but for this I will have to enhance the functionality of `get_feature_importances` so that it returns a dictionary of {feature_name: relative feature importance} rather than just the latter.